### PR TITLE
Add paasta deploy events using yelp_meteorite.

### PR DESCRIPTION
**Pull request by rahulravindran0108**
**Summary**
This mimics the internal functionality for event_detector which was
recently decommissioned. Emits an event name-spaced `deploy.paasta`
and along with this sends cluster, instance and service as a dimension.
Open to instrument more dimensions if need be.

### CODE REVIEW

File: paasta_tools/setup_marathon_job.py [hunk 1]
```diff
@@ -392,6 +399,17 @@ def log_bounce_action(line: str, level: str='debug') -> None:
                 ),
                 level='event',
             )
+
+            if yelp_meteorite:
+                # Emit deploy.paasta event
+                yelp_meteorite.events.emit_event(
+                    'deploy.paasta',
+                    dimensions={
+                        'cluster': cluster,
```
Comment (solarkennedy) [inline paasta_tools/setup_marathon_job.py]: Here is some bad news: we haven't been consistent in our dimensions with our metrics.
https://github.com/Yelp/paasta/blob/master/paasta_tools/contrib/emit_allocated_cpu_metrics.py#L26

What do you think about getting us off onto the "right" foot and moving us twoards a consistent `paasta_cluster`, `paasta_service` and `paasta_instance` ?

File: paasta_tools/setup_marathon_job.py [hunk 2]
```diff
@@ -392,6 +399,17 @@ def log_bounce_action(line: str, level: str='debug') -> None:
                 ),
                 level='event',
             )
+
+            if yelp_meteorite:
+                # Emit deploy.paasta event
```
Comment (solarkennedy) [inline paasta_tools/setup_marathon_job.py]: This comment doesn't really say anything that the next line doesn't say

File: paasta_tools/setup_marathon_job.py (hunk 1 above)
Comment (rahulravindran0108) [reply paasta_tools/setup_marathon_job.py]: I'll go ahead and rename these dimensions to be consistent with others.

File: paasta_tools/setup_marathon_job.py (hunk 2 above)
Comment (rahulravindran0108) [reply paasta_tools/setup_marathon_job.py]: Agreed. Will remove this

**Comment by rahulravindran0108** [comment]
@solarkennedy addressed inconsistencies with dimensions. This is ready for review.

**Pull request merged**